assets/js/modules/idea-hub/components/dashboard/DashboardCTA.js

```diff
@@ -70,8 +90,17 @@ function DashboardCTA( { Widget, WidgetNull } ) {
 		}
 	}, [ activateModule, navigateTo, setInternalServerError ] );
 
+	const onLinkClick = useCallback( () => {
+		trackEvent( IDEA_HUB_GA_CATEGORY_WIDGET, 'click_outgoing_link' );
```

Comment:
This event is missing the event label from the ACs.

Reply:
Updated.